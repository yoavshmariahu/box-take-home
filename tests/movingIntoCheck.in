k a1
K e5
p a2
R e2

[G S B R]
[p b s g]

move a1 b2
