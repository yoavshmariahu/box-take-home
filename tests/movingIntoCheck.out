lower player action: move a1 b2
5 |__|__|__|__| K|
4 |__|__|__|__|__|
3 |__|__|__|__|__|
2 | p|__|__|__| R|
1 | k|__|__|__|__|
    a  b  c  d  e

Captures UPPER: G S B R
Captures lower: p b s g

UPPER player wins.  Illegal move.
